api/src/main/java/com/datastrato/gravitino/rel/types/Types.java

```diff
@@ -34,6 +34,28 @@ public String simpleString() {
     }
   }
 
+  /** The data type representing `UNKNOWN` values. */
+  public static class UnknownType implements Type {
+    private static final UnknownType INSTANCE = new UnknownType();
+
+    /** @return The singleton instance of {@link UnknownType}. */
+    public static UnknownType get() {
+      return INSTANCE;
+    }
+
+    private UnknownType() {}
+
+    @Override
+    public Name name() {
+      return Name.UNKNOWN;
+    }
+
+    @Override
+    public String simpleString() {
```

Comment:
The `Unknown` type should retain the original type information, so that users can understand the actual type that cannot be processed through the `Unknown`  type. 

For example, the `bit` type in MySQL can be mapped to Gravitino's `Unknown(bit)`.